openmetadata-ui/src/main/resources/ui/src/components/NavBar/NavBar.tsx

```diff
@@ -325,7 +319,27 @@ const NavBar = () => {
       ) {
         return;
       }
+
+      // Check if we need to fetch based on cache timing
+      // This is to block the API call for 1 hour
+      const lastFetchTime = cookieStorage.getItem(LAST_VERSION_FETCH_TIME_KEY);
+      const now = Date.now();
+
+      if (lastFetchTime) {
+        const timeSinceLastFetch = now - parseInt(lastFetchTime);
+        if (timeSinceLastFetch < ONE_HOUR_MS) {
+          // Less than 1 hour since last fetch, skip API call
+          return;
+        }
+      }
+
       const newVersion = await getVersion();
+
+      // Update the cache timestamp
+      cookieStorage.setItem(LAST_VERSION_FETCH_TIME_KEY, String(now), {
+        expires: new Date(Date.now() + ONE_HOUR_MS),
+      });
+
```

Comment:
shouldn't you be calling fetchOMVersion here ?

Reply:
No as there we are updating version to state, we want to show banner upon version missmatch hence we have added seprate method here